react-native/todo/src/components/Todos/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+
+import AddTodo from './components/AddTodo';
+import TodoList from './components/TodoList';
+
+class Todos extends Component {
+  render() {
+    return (
+      <Fragment>
```

Comment:
Which version of React are we using ? Notice that this only works if `React >= 16.2`

Reply:
16.2 :smiley: